### ISSUE TITLE
naming-convention rule bug - single letter field name

### ISSUE THREAD

**Issue by joshribakoff-sm** [B2o5T/graphql-eslint#1007]
**Describe the bug**
I have a single letter field name, it is lowercase which I consider valid snake case.

**To Reproduce**

```json
"@graphql-eslint/naming-convention": [
          "error",
          {
            "FieldDefinition": "snake_case",
            "InputValueDefinition": "snake_case",
            "VariableDefinition": "snake_case"
          }
        ]
```

```gql
type Point {
  x: String
  y: String
}

```

**Expected behavior**
It should not be an error, as this otherwise is valid snake case

**Environment:**

- OS: mac
- `@graphql-eslint/...`: 3.8.0
- NodeJS: 16

**Additional context**
If I make the name 2 characters long the rule considers it okay.

**Comment by B2o5T** [comment]
Hi @joshribakoff-sm, thank you for your report, saw this bug also for `UPPER_CASE` style, this will be fixed on the next version of `graphql-eslint` 🚀. Added related tests also

**Comment by joshribakoff-sm** [comment]
Thanks!!

**Comment by B2o5T** [comment]
fixed in [@graphql-eslint/eslint-plugin@3.10.0](https://github.com/B2o5T/graphql-eslint/releases/tag/%40graphql-eslint%2Feslint-plugin%403.10.0)
